v2/api/resources.md

```diff
@@ -43,6 +43,17 @@ Use CRD version `v1alpha1api20201201`
 
 Use CRD version `v1alpha1api20210501`
 
+### microsoft.dbformysql
+
+
+#### ARM version 2021-05-01
+
+- FlexibleServer ([sample](https://github.com/Azure/azure-service-operator/tree/main/v2/config/samples/microsoft.dbformysql/v1alpha1api20210501_flexibleserver.yaml))
+- FlexibleServersDatabase ([sample](https://github.com/Azure/azure-service-operator/tree/main/v2/config/samples/microsoft.dbformysql/v1alpha1api20210501_flexibleserversdatabase.yaml))
```

Comment:
Is there a corresponding `FlexibleServerConfiguration` which we also added support for in Postgres?

Reply:
No, I looked for that but there isn't a corresponding one in the MySQL resources:
https://docs.microsoft.com/en-us/azure/templates/microsoft.dbformysql/allversions
vs
https://docs.microsoft.com/en-us/azure/templates/microsoft.dbforpostgresql/allversions